packages/cloud-cognitive/src/components/AddSelect/AddSelectSidebar.js

```diff
@@ -29,9 +29,14 @@ export let AddSelectSidebar = ({
     setMultiSelection(newSelections);
   };
 
-  const sidebarItems = multiSelection.map((selectedId) =>
-    items.find((item) => item.id === selectedId)
-  );
+  const sidebarItems = multiSelection.reduce((acc, cur) => {
+    const i = items.find((item) => item.id === cur);
```

Comment:
Can we name this variable something more specific?

Reply:
done